### ISSUE TITLE
Identifying non-data columns

### ISSUE THREAD

**Issue by mattansb** [easystats/parameters#1023]
In one of the recent `bayestestR` PRs (https://github.com/easystats/bayestestR/pull/673 & https://github.com/easystats/bayestestR/pull/672) some of the printing methods have adopted allowing for arbitrary columns in the resulting data frame objects. This was done by setting a new type of attribute called `idvars` the contained the names of the columns that don't hold the "statistical" information, but information used to identify the rows.

I've found this to better and more stable than keeping track of all possible column names, and very flexible.

I wonder if this should be used across `easystats`? We can have several "classes" of such attributes - `idvars`, grouping-vars, etc....

These could be useful if "detected" by the various formatting and printing methods in `parameters` (which is why I've opened this issue here, but feel free to move it elsewhere) and in `insight` (maybe also `datawizard`, `correlation` and `modelbased`?).

WDYT? @easystats/core-team

**Comment by strengejacke** [comment]
Do you have an example that makes clear how this affects printing or how we have to change methods?

**Comment by strengejacke** [comment]
ok, I see. I think this is something that needs to be handled in the packages' `format()` methods - in _insight_, only the "final" data frame is processed, no filtering/column-selection is usually done there.

We should then decide on the attributes' names. If I look at your code changes, you would suggest the attribute `idvars` for those columns that should also be included in the output, additional to the default-columns, right?

**Comment by mattansb** [comment]
>  I think this is something that needs to be handled in the packages' format() methods

Yes, the way things are setup now. But perhaps this can be directly adapted into `insight::format_table()` or `insight::export_table()` at some point.
